docs/dev.md

```diff
@@ -0,0 +1,60 @@
+Development
+===========
+This page contains information for developers about how to contribute to this project.
+
+Set-up
+------
+This project can be used in combination with  [dans-dev-tools]{:target=_blank}. Before you can start it as a service
+some dependencies must first be started:
+
+### Environment
+
+Open a separate terminal tab for `dd-transfer-to-vault` and one for its dependency `dd-vault-catalog` and run:
+
+```commandline
+start-env.sh
+```
+
+### Database
+
```

Comment:
Why not set the DB port to 9002 in one of the two project's `config.yml` and `db.properties`?